### ISSUE TITLE
stockPositiveValue and stockNegativeValue don't affect how chart looks.

### ISSUE THREAD

**Issue by InnerDesires** [amcharts/amcharts5#728]
I've seen in your recent update log that there was an issue with these properties not being inherited right, however, I'm stuck with changing the colors of my candlestick diagram.
I'm learning how to use amCharts with React, the version of amcharts being used is: 
```
    "@amcharts/amcharts5": "^5.2.45",
     "react": "^18.2.0"
```
I'm setting those props like this: 
```
let stockChart = root.container.children.push(
            am5stock.StockChart.new(root, {
                layout: root.verticalLayout,
                stockPositiveColor: am5.color('#00f'),
                stockNegativeColor: am5.color('#00f')
            })
        );
        root.numberFormatter.set("numberFormat", "#,###.00");
        let mainPanel = stockChart.panels.push(am5stock.StockPanel.new(root, {
            wheelY: "zoomX",
            panX: true,
            panY: true,
        }));
```

However, the look of my chart is not being affected.

Also, the theming approach doesn't seem to work too: 
```
let root = am5.Root.new("chartdiv");
        let myTheme = am5.Theme.new(root);
        myTheme.rule('StockChart').setAll({
            'stockPositiveColor': am5.color('#00F'),
            'stockNegativeColor': am5.color('#00F'),
            'volumePositiveColor': am5.color('#00F'),
            'volumeNegativeColor': am5.color('#00F'),
        })
```
The way the chart looks: 
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/14187425/208689140-0d462cbb-f745-4c91-91b1-a061fe8a31b7.png">

**Comment by martynasma** [comment]
These settings apply to main value and volume series only. If you need your series to have these colors applied, set it as `stockSeries` on `StockChart`:

https://www.amcharts.com/docs/v5/charts/stock/#Positive_negative_colors

**Comment by MementoMori78** [comment]
@martynasma 
> https://www.amcharts.com/docs/v5/charts/stock/#Positive_negative_colors
Thanks for the reply, but I've already visited this part of your docs several times today and was setting those styles according to it:
```
let stockChart = root.container.children.push(
            am5stock.StockChart.new(root, {
                layout: root.verticalLayout,
                stockPositiveColor: am5.color(0x999999),
                stockNegativeColor: am5.color(0x000000),
                volumePositiveColor: am5.color(0x999999),
                volumeNegativeColor: am5.color(0x000000)
            })
        );
...  
stockChart.set("stockSeries", valueSeries);

```
And as you can see on the screenshot, only the volume series is actually being affected by the passed props:
![image](https://user-images.githubusercontent.com/14187425/208748731-e60ff9b1-c8fb-4263-a8cf-86ee7acd4423.png)

**Comment by zeroin** [comment]
There is a bug on our side, we will fix it in next release and will let you know. Meanwhile the quick fix would be to modify interface colors:
```
root.interfaceColors.set("positive", am5.color(0x999999));
root.interfaceColors.set("negative", am5.color(0x000000));
```

**Comment by martynasma** [comment]
Fixed in [5.2.46](https://github.com/amcharts/amcharts5/releases/tag/5.2.46).

## [5.2.46] - 2023-01-02

### Fixed
- Setting `stockPositiveColor` and `stockNegativeColor` initially on a `StockChart` was not affecting colors of candlesticks.
- When `Tooltip` had `keepTargetHover: true` set, the portions of the tooltip outside the chart area were not being registered for hovers when the `Root` element had `tooltipContainerBounds` set.
- `DateAxis` with data grouping enabled could go into infinite loop of switching between period in some specific data setups.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
